sys-apps/diskus/diskus-0.4.0.recipe

```diff
@@ -0,0 +1,237 @@
+
+SOURCE_URI_42="https://static.crates.io/crates/strsim/strsim-0.7.0.crate"
+CHECKSUM_SHA256_42="bb4f380125926a99e52bc279241539c018323fab05ad6368b56f93d9369ff550"
+
+SOURCE_URI_43="https://static.crates.io/crates/term_size/term_size-0.3.1.crate"
+CHECKSUM_SHA256_43="9e5b9a66db815dcfd2da92db471106457082577c3c278d4138ab3e3b4e189327"
+
+SOURCE_URI_44="https://static.crates.io/crates/termion/termion-1.5.1.crate"
+CHECKSUM_SHA256_44="689a3bdfaab439fd92bc87df5c4c78417d3cbe537487274e9b0b2dce76e92096"
+
+SOURCE_URI_45="https://static.crates.io/crates/textwrap/textwrap-0.10.0.crate"
+CHECKSUM_SHA256_45="307686869c93e71f94da64286f9a9524c0f308a9e1c87a583de8e9c9039ad3f6"
+
+SOURCE_URI_46="https://static.crates.io/crates/unicode-width/unicode-width-0.1.5.crate"
+CHECKSUM_SHA256_46="882386231c45df4700b275c7ff55b6f3698780a650026380e72dabe76fa46526"
+
+SOURCE_URI_47="https://static.crates.io/crates/unreachable/unreachable-1.0.0.crate"
+CHECKSUM_SHA256_47="382810877fe448991dfc7f0dd6e3ae5d58088fd0ea5e35189655f84e6814fa56"
+
+SOURCE_URI_48="https://static.crates.io/crates/vec_map/vec_map-0.8.1.crate"
+CHECKSUM_SHA256_48="05c78687fb1a80548ae3250346c3db86a80a7cdd77bda190189f2d0a0987c81a"
+
+SOURCE_URI_49="https://static.crates.io/crates/version_check/version_check-0.1.5.crate"
+CHECKSUM_SHA256_49="914b1a6776c4c929a602fafd8bc742e06365d4bcbe48c30f9cca5824f70dc9dd"
+
+SOURCE_URI_50="https://static.crates.io/crates/void/void-1.0.2.crate"
+CHECKSUM_SHA256_50="6a02e4885ed3bc0f2de90ea6dd45ebcbb66dacffe03547fadbb0eeae2770887d"
+
+SOURCE_URI_51="https://static.crates.io/crates/winapi/winapi-0.2.8.crate"
+CHECKSUM_SHA256_51="167dc9d6949a9b857f3451275e911c3f44255842c1f7a76f33c55103a909087a"
+
+SOURCE_URI_52="https://static.crates.io/crates/winapi/winapi-0.3.6.crate"
+CHECKSUM_SHA256_52="92c1eb33641e276cfa214a0522acad57be5c56b10cb348b3c5117db75f3ac4b0"
+
+SOURCE_URI_53="https://static.crates.io/crates/winapi-build/winapi-build-0.1.1.crate"
+CHECKSUM_SHA256_53="2d315eee3b34aca4797b2da6b13ed88266e6d612562a0c46390af8299fc699bc"
+
+SOURCE_URI_54="https://static.crates.io/crates/winapi-i686-pc-windows-gnu/winapi-i686-pc-windows-gnu-0.4.0.crate"
+CHECKSUM_SHA256_54="ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6"
+
+SOURCE_URI_55="https://static.crates.io/crates/winapi-x86_64-pc-windows-gnu/winapi-x86_64-pc-windows-gnu-0.4.0.crate"
+CHECKSUM_SHA256_55="712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	diskus$secondaryArchSuffix = $portVersion
+	cmd:diskus
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cargo$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage diskus$secondaryArchSuffix \
+	$binDir/diskus
```

Comment:
```suggestion
	$commandBinDir/diskus
```

Reply:
That's what I get for manually replacing them... thanks!